packages/react-label/src/components/Label/Label.types.ts

```diff
@@ -32,6 +32,13 @@ export interface LabelProps extends ComponentProps, React.LabelHTMLAttributes<HT
   strong?: boolean;
 }
 
+/**
+ * (Internal) LabelProps after the required prop has been resolved from a boolean to a string
+ */
+export interface LabelPropsResolved extends LabelProps {
```

Comment:
If this is internal but you want to export it for use in other files you should look into individually exporting the other things in  `index.ts` so this is not available outside this package.

Reply:
I think this would need to be exported since it's being referenced by the exported `LabelState` type. That said, it's a bit hacky--I'll refactor it to not need this intermediate type.